.github/workflows/release.yml

```diff
@@ -156,3 +156,9 @@ jobs:
           files: |
             phylum-*.zip
             phylum-*.zip.minisig
+
+  Deploy-Docs:
+    name: Deploy Docs
```

Comment:
"Publish" or "Update" might be slightly better than "Deploy" and more in line with the notion of documentation.